coil/src/main/java/dev/chrisbanes/accompanist/coil/Coil.kt

```diff
@@ -43,6 +44,21 @@ import dev.chrisbanes.accompanist.imageloading.ImageLoadState
 import dev.chrisbanes.accompanist.imageloading.MaterialLoadingImage
 import dev.chrisbanes.accompanist.imageloading.toPainter
 
+/**
+ * Ambient containing the preferred [ImageLoader] to use in [CoilImage].
+ */
+val AmbientImageLoader = staticAmbientOf<ImageLoader?> { null }
+
+object CoilImageConstants {
```

Comment:
Constants or defaults?

Reply:
I'm sticking with whatever Material does for now for consistency.